### PR TITLE
updated snowplow js file

### PR DESCRIPTION
**Pull request by skiselev1**
# Description

This PR includes the following proposed change(s):

- updated snowplow js file

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/csrs-portal/src/app/components/child-application-question/child-application-question.component.ts [hunk 1]
```diff
@@ -59,6 +60,8 @@ export class ChildApplicationQuestionComponent implements OnInit {
...
...mlhdCI6MTY0NDQzMjc1MiwiYXV0aF90aW1lIjoxNjQ0NDMxMTgyLCJqdGkiOiI0MzlhNzFmMS0xOWE5LTQ4NGQtYjM2Ni1lMWFlMjU2NWUzNWIiLCJpc3MiOiJodHRwczovL2Rldi5vaWRjLmdvdi5iYy5jYS9hdXRoL3JlYWxtcy9vbmVzdG9wYXV0aC1iYXNpYyIsImF1ZCI6ImpzYi1jc3JzLTk5MSIsInN1YiI6IjY1YjAzNDI0LWUyNTQtNDhiNy04NDAzLWM3MTNjMGQ1ODQ0MyIsInR5cCI6IkJlYXJlciIsImF6cCI6ImpzYi1jc3JzLTk5MSIsIm5vbmNlIjoiZTExZjBjNTM1NDA5MzE1MGQ5NGFmMWYyZWIxYjAzNTFjMkF0Ym00Y0EiLCJzZXNzaW9uX3N0YXRlIjoiODY1ZjYwNGItY2ExYS00ZmFhLTg0NDgtNjRhZmU3ZWE4ODVjIiwiYWNyIjoiMCIsImFsbG93ZWQtb3JpZ2lucyI6WyJodHRwczovL2Rldi5jaGlsZHN1cHBvcnRyZWNhbGMuZ292LmJjLmNhIl0sInNjb3BlIjoib3BlbmlkIGVtYWlsIHByb2ZpbGUiLCJpZGVudGl0eV9wcm92aWRlciI6ImJjZWlkLWJhc2ljIiwiZW1haWxfdmVyaWZpZWQiOmZhbHNlLCJiY2VpZF91c2VyaWQiOiIyNjMzNjA3MkNCQTQ0QjZFODcxQjUzNTVEMjdERjlCMyIsInByZWZlcnJlZF91c2VybmFtZSI6Imhhbjk5QGJjZWlkLWJhc2ljIiwiZW1haWwiOiJoYW5zb2xvQHN0YXJ3YXJzLmNvbSJ9.MEDx__kEKNrsvLGImvlyfEs4l3g7qtwK_byMOOGTa6kQ5TyrMBmFBbGKzi-b4XCDSAU1AH8UGH23vm62aADqzDECC7BJgBtGK-xy3jw_SNviS0okFup8BhFG23llpvRRPCxyxKPYgpdzGg2q2OZdBSAHWLFt8PIQWGcQpvG7pAiI-zASK6tZNdWpKB13T2OTE1jWOEVlvAsVaRc2-ys5FVe773NihIrpuWdU9CRHB8PbFLjCYNNo6sR7RIT7fhi4RafzTQ-r9ZKpSKmY-Uav1INeLj3Xnn6mvIy3OwBrcenYoLAgACi0qRTrqGo1cPBjvqboYGLXdOCL6uHp7UKhng';
```
Comment (TayGov) [inline src/frontend/csrs-portal/src/app/components/child-application-question/child-application-question.component.ts:63]: Nothing major but should we keep this?

**Review by TayGov — CHANGES_REQUESTED** [review]
Looks good one comment 👍

File: src/frontend/csrs-portal/src/app/components/child-application-question/child-application-question.component.ts (hunk 1 above)
Comment (skiselev1) [reply src/frontend/csrs-portal/src/app/components/child-application-question/child-application-question.component.ts]: Thank you! You are right! it must be removed! :)

**Review by TayGov — APPROVED** [review]
👍

**Pull request merged**